src/Makefile

```diff
@@ -77,6 +81,16 @@ $(LIBDIR)/$(STATICLIBTARGET): $(LIBOBJ) $(DEVICELIB)
 	ar cr $@ $(LIBOBJ) $(TMP)/*.o
 	rm -Rf $(TMP)
 
+$(PKGDIR)/nccl.pc : nccl.pc.in
+	@mkdir -p $(PKGDIR)
+	printf "Generating %-35s > %s\n" $< $@
+	printf $(PREFIX)
+	sed -e 's|$${nccl:Prefix}|\$(PREFIX)|g' \
```

Comment:
Is there a reason for this sed expr being different from the others below ? Note I don't have a preference, just looking for consistency.

Reply:
because i test with other form and sed fails, i think because the '$(PREFIX)', the start character is '/'

Reply:
the generated pc file is:

~~~
prefix=/usr/local
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: nccl
Description: Optimized primitives for collective multi-GPU communication
Version: 2.4.2
Libs: -L${libdir} -lnccl
Cflags: -I${includedir}
~~~
(makefile launched with: make CUDA_HOME='/opt/cuda' -j40 V=1)